test/helper.ts

```diff
@@ -33,3 +35,17 @@ export function parseAndProcess(input: string, fn: (ast: ESTree.Program) => void
   fn(ast);
   return astring(ast).trim();
 }
+
+const defaultSandbox = {
+  console,
+  module: {
+    exports: {}
+  },
+  require
+};
+
+export function virtualModule(code: string, optionsSandbox = {}) {
```

Comment:
If it was intended, then okay.
Personally I don't like this kind of 'open interfaces'. Its a too wide interface and could introduce much missusage and bugs.